app/src/main/java/fr/paug/androidmakers/ui/components/AgendaLayout.kt

```diff
@@ -96,6 +115,16 @@ private fun AgendaPagerOrLoading(
     }
 }
 
+/** Returns the index of today's [DaySchedule] in `this`, or zero. */
+private fun List<DaySchedule>.todayPageIndex(): Int{
+    val todayOfYear = Instant.now().atZone(ZoneId.systemDefault()).dayOfYear
+    val todayPageIndex = indexOfFirst { (dayInMillis, _, _) ->
+        val dayOfYear = Instant.ofEpochMilli(dayInMillis).atZone(ZoneId.systemDefault()).dayOfYear
+        todayOfYear == dayOfYear
+    }
+    return if (todayPageIndex < 0) 0 else todayPageIndex
```

Comment:
Time is hard 😅 . Do we really need to go through timezones here since will have epochMillis everywhere? Maybe something like:

```kotlin
private fun List<DaySchedule>.todayPageIndex(): Int{
    val todayPageIndex = indexOfLast { (dayInMillis, _, _) ->
        System.currentTimeMillis() >= dayInMillis
    }
    return if (todayPageIndex < 0) 0 else todayPageIndex
}
```
?

Reply:
fixed